tools/versionbump/versionbump.go

```diff
@@ -0,0 +1,83 @@
+package main
```

Comment:
Tito's version tagger doesn't need to be told what version, by default it just bumps whatever is in the spec at that time.